shell/shared/admin.js

```diff
@@ -409,7 +409,16 @@ if (Meteor.isClient) {
     },
     userIsGuest: function () {
       return !this.isAdmin && !this.signupKey;
-    }
+    },
+    userQuota: function () {
+      return (typeof this.quota === "number") ? prettySize(this.quota) : "";
```

Comment:
Eh, I feel it's more readable to have people with no quota show up blank in the table. Easier to skim through and find the people who do have quota (or vice versa).